src/annotation/annotation.test.ts

```diff
@@ -47,6 +47,7 @@ const spline: Spline = {
   ],
   spaceTimeInfo: { z: 0, t: 0 },
   isClosed: false,
+  isBezier: false,
```

Comment:
Yeah, I think this is something to sort out when we centralise interfaces and import them into the SDK. The migration system doesn't currently handle annotation or audit content interfaces because those interfaces are defined in other repos, but one day they'll be centralised in their own repo and then it'll make more sense to bring them into the migration system. Until then, I think this particular flag shouldn't cause problems if it's missing - `undefined` is falsy so if it's missing that should be interpreted as not Bezier. For now though I'll put another migration in `DominateStore.correctPluginModifications`, that'll add the field if it's missing.